packages/checkout/widgets-lib/src/widgets/swap/helpers.ts

```diff
@@ -1,6 +1,5 @@
 import { TokenInfo, IMTBLWidgetEvents } from '@imtbl/checkout-sdk';
-import { text } from '../../resources/text/textConfig';
-import { SharedViews } from '../../context/view-context/ViewContext';
+import { useTranslation } from 'react-i18next';
```

Comment:
will need to change how this is handled in the ts files as cant use hooks outside of react components